staging/src/k8s.io/api/coordination/v1/types.go

```diff
@@ -58,6 +74,14 @@ type LeaseSpec struct {
 	// holders.
 	// +optional
 	LeaseTransitions *int32 `json:"leaseTransitions,omitempty" protobuf:"varint,5,opt,name=leaseTransitions"`
+	// Strategy indicates the strategy for picking the leader for coordinated leader election
+	// (Alpha) Using this field requires the CoordinatedLeaderElection feature gate to be enabled.
+	// +featureGate=CoordinatedLeaderElection
+	// +optional
```

Comment:
I think you also want `+default="OldestCompatibilityVersion"` don't you?

Reply:
No changes should be required here for leaving the field blank. If CLE created the Lease, it can set the OldestCompatibilityVersion explicitly, and if a legacy candidate created the lease, it should be empty.

Reply:
If that's the way we go, then we do not need a const for "NoCoordination"

Reply:
Before we lock that in: Will there be any client confusion wrt version skew?  I don't see any, but I will think about it more

Reply:
I am un-resolving just to be sure we don't have skew issues

Reply:
This entire KEP is targeting version skew so I'm hoping it does. A legacy candidate will try to obtain the lease lock without any watch/other mechanisms. CLE controller will only elect a new leader if the current holder is empty (since legacy candidates are not elected by CLE). In mixed verison state, there will be two behaviors depending on who the lock holder is.

1. If CLE elected candidate holds the lock, it will continue to renew the lease. Legacy candidates will not publish LeaseCandidate objects and will not be in consideration by CLE.
2. If a legacy candidate holds the lock, anything that CLE does will be a noop since the candidate doesn't know how to ack a PreferredHolder transition.
3. If a lease holder goes away, other legacy candidates are racing for the lock. CLE is also watching the lock. If legacy candidates win the race, we go to case 1. If CLE wins the race we go to case 2.

Reply:
I'm not clear if you are stasting that there's a problem or that it is designed as intended?

Reply:
Trying to say that this is handled as intended.